#include "task_exception.h"

task_exception::task_exception()
{

}
